firedrake/slate/slac/kernel_builder.py

```diff
@@ -737,7 +738,7 @@ def generate_tsfc_calls(self, terminal, loopy_tensor):
             for tsfc_kernel in cxt_kernel.tsfc_kernels:
                 integral_type = cxt_kernel.original_integral_type
                 slate_tensor = cxt_kernel.tensor
-                mesh = slate_tensor.ufl_domain()
+                mesh = extract_unique_domain(slate_tensor)
```

Comment:
I think all of the changes in this file need to be reverted.

Reply:
reverted